frontend/app/src/components/dashboard/edit-snapshot/EditBalancesSnapshotForm.vue

```diff
@@ -4,13 +4,11 @@ import { toMessages } from '@/utils/validation';
 import { isNft } from '@/utils/nft';
 import { usePropVModel } from '@/utils/model';
 import { useEditBalancesSnapshotForm } from '@/composables/snapshots/edit-balance/form';
-import EditBalancesSnapshotLocationSelector
-  from '@/components/dashboard/edit-snapshot/EditBalancesSnapshotLocationSelector.vue';
 import BalanceTypeInput from '@/components/inputs/BalanceTypeInput.vue';
+import EditBalancesSnapshotLocationSelector from './EditBalancesSnapshotLocationSelector.vue';
+import EditBalancesSnapshotAssetPriceForm from './EditBalancesSnapshotAssetPriceForm.vue';
```

Comment:
Regex was not perfect, I found a couple of cases that I fixed when adjusting the imports.

Can we do a full path alias import in both cases please?